### DISCUSSION TITLE
Collaboration with Penpot?

### DISCUSSION THREAD

**Discussion by JohnDumpling** [akiraux/Akira#723] [General]
Have you heard about Penpot? It is a web-based, open-source, UI and UX design app. Here is a demo:
![obrázok](https://user-images.githubusercontent.com/89086803/167311822-d89a9cc6-299d-46f3-9d45-03e803a85708.png)

They have already developed a lot of important features needed for designing user interfaces, maybe this project could be an inspiration and a great help for Akira. Link to their website: https://penpot.app/

**Comment by Alecaddd** [comment]
I converted this into a discussion since it's not really an "issue" per se.
I'm aware of PenPot, in fact, Akira and PenPot kinda started at the same time, when PenPot was first called UXBox.
Their stack is pure web and based on Clojure, so not compatible with the native GTK stack of Akira.

PenPot is more focused on collaboration and cloud prototyping, as for now Akira is focused on local UX/UI design.
All the features that PenPot comes with will sooner or later be available in Akira, as those features are very much standards in modern UX design app (Figma, Sketch, Adobe XD, etc).

Once Akira is in a stable state, I would love to kick off a collaboration with PenPot to allow importing and exporting files between apps.
